src/java/org/apache/cassandra/service/StorageProxy.java

```diff
@@ -1450,6 +1450,7 @@ private static List<Row> fetchRows(List<ReadCommand> initialCommands, Consistenc
                 try
                 {
                     Row row = exec.get();
+                    Keyspace.open(exec.command.ksName).getColumnFamilyStore(exec.command.cfName).metric.avoidedReadRepairs.mark();
```

Comment:
For future ref, this metric is placed here because exec.get() will throw a `DigestMismatchException` if values do not agree